koala/ast/tokenizer.py

```diff
@@ -388,9 +388,9 @@ def EOF():
             # standard postfix operators
             if ("%".find(currentChar()) != -1):
                 if (len(token) > 0):
-                    tokens.add(token, self.TOK_TYPE_OPERAND)
+                    tokens.add(float(token) / 100, self.TOK_TYPE_OPERAND)
```

Comment:
See https://github.com/iOiurson/koala/commit/c55b9ee00a561255bcd47c518fc1829031036500.

Values with `a%` were considered as `a` instead of `0.0a`